lib/elixir/lib/supervisor.ex

```diff
@@ -638,7 +654,15 @@ defmodule Supervisor do
   description of the available strategies.
   """
   @doc since: "1.5.0"
-  @spec init([:supervisor.child_spec() | {module, term} | module], [init_option]) :: {:ok, tuple}
+  @spec init(
+          [
+            child_spec()
+            | {module, term}
+            | module
+            | (old_erlang_child_spec :: :supervisor.child_spec())
+          ],
+          [init_option]
+        ) :: {:ok, sup_flags() | (old_erlang_sup_flags :: :supervisor.sup_flags())}
```

Comment:
It does not return old_erlang_sup_flags. :)

Reply:
fixed